src/domain/balances/balances.repository.ts

```diff
@@ -66,16 +110,6 @@ export class BalancesRepository implements IBalancesRepository {
     );
   }
 
-  async clearLocalBalances(args: {
```

Comment:
This function was moved upward to keep class public functions/interface members first, and private/utility functions later.